### PR TITLE
Random doc typos

### PR DESCRIPTION
**Pull request by ferruzzi**
---
**^ Add meaningful description above**
Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/contributing-docs/05_pull_requests.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

File: contributing-docs/testing/unit_tests.rst [hunk 1]
```diff
@@ -459,7 +459,7 @@ Problems with Non-DB test collection
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Sometimes, even if whole module is marked as ``@pytest.mark.db_test`` even parsing the file and collecting
-tests will fail when ``--skip-db-tests`` is used because some of the imports od objects created in the
+tests will fail when ``--skip-db-tests`` is used because some of the imports or objects created in the
```
Comment (ferruzzi) [inline contributing-docs/testing/unit_tests.rst:462]: Maybe this was supposed to be "of"?

File: contributing-docs/testing/unit_tests.rst [hunk 2]
```diff
@@ -338,7 +338,7 @@ If your test accesses the database but is not marked properly the Non-DB test in
 How to verify if DB test is correctly classified
 ................................................
 
-When you add if you want to see if your DB test is correctly classified, you can run the test or group
+If you want to see if your DB test is correctly classified, you can run the test or group
 of tests with ``--skip-db-tests`` flag.
 
 You can run the all (or subset of) test types if you want to make sure all ot the problems are fixed
```
Comment (shahar1) [inline contributing-docs/testing/unit_tests.rst]:
```suggestion
You can run the all (or subset of) test types if you want to make sure all of the problems are fixed
```

Comment (shahar1) [reply contributing-docs/testing/unit_tests.rst]: That's certainly an `of` :)

File: contributing-docs/testing/unit_tests.rst [hunk 3]
```diff
@@ -459,7 +459,7 @@ Problems with Non-DB test collection
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Sometimes, even if whole module is marked as ``@pytest.mark.db_test`` even parsing the file and collecting
```
Comment (shahar1) [inline contributing-docs/testing/unit_tests.rst]:
```suggestion
Sometimes, even if whole module is marked as ``@pytest.mark.db_test``, parsing the file and collecting
```

File: contributing-docs/testing/unit_tests.rst (hunk 1 above)
Comment (shahar1) [reply contributing-docs/testing/unit_tests.rst:462]: I'd go with `of`

**Review by shahar1 — APPROVED** [review]
Feel free to merge after resolving :)

**Review by amoghrajesh — APPROVED** [review]

File: contributing-docs/testing/unit_tests.rst (hunk 3 above)
Comment (ferruzzi) [inline contributing-docs/testing/unit_tests.rst]:
```suggestion
Sometimes, even if the whole module is marked as ``@pytest.mark.db_test``, parsing the file and collecting
```

**Pull request merged**